pkg/utils/common/service_account.go

```diff
@@ -39,3 +39,13 @@ func (s *SuiteController) CreateServiceAccount(name, namespace string, serviceAc
 func (h *SuiteController) DeleteAllServiceAccountsInASpecificNamespace(namespace string) error {
 	return h.KubeRest().DeleteAllOf(context.TODO(), &corev1.ServiceAccount{}, client.InNamespace(namespace))
 }
+
+// DeleteServiceAccount deletes a service account with the provided name and namespace
+func (s *SuiteController) DeleteServiceAccount(name, namespace string) error {
+	// if ServiceAccount doesn't exist, return nil
+	_, err := s.GetServiceAccount(name, namespace)
+	if err != nil {
```

Comment:
Same as above. To ensure that the ServiceAccount was not found, we should check `err.IsNotFound`, imho. WDYT?

Reply:
thanks for your suggestion. fixed